the-blue-alliance-ios/Core Data/Match.swift

```diff
@@ -53,15 +53,15 @@ extension Match: Managed {
         }
         switch compLevel {
         case MatchCompLevel.qualification.rawValue:
-            return "Qual"
+            return "Quals"
         case MatchCompLevel.eightfinal.rawValue:
-            return "Octofinal"
```

Comment:
We want these strings to match Android/web, so this might roll back. Adding a pluralization seems right to me... probably something I’ll check out when I build

Reply:
Oh - the web changed this 😅 Yeah let's go ahead and keep this then